src/test/java/org/mockito/internal/framework/DefaultMockitoFrameworkTest.java

```diff
@@ -112,5 +123,43 @@ public void run() {
                     "For more information, see the javadoc for RedundantListenerException class.");
     }
 
+    @Test
+    public void clears_all_mocks() {
+        //clearing mocks only works with inline mocking
+        assumeTrue(Plugins.getMockMaker() instanceof InlineMockMaker);
```

Comment:
FWIW, one of the reason why I didn't use assume at first is that it can test user can safely call clearInlineMocks() and clearMock() even when mock maker is not a inline mock maker.

That's not very important I suppose, but still would be good to cover that in tests imo.

Reply:
+1 I'm fixing it now.